src/components/EmptyState/Actions.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import * as Rivet from "../util/Rivet";
+
+const Actions = ({ children, ...attrs }) => (
+  <div {...attrs} className="rvt-empty-state__actions">
```

Comment:
Should `className` merge (rather than replace) classes brought in with `attrs.className`? Is this the pattern done with other components?

Reply:
Good question, I'll need to look how we're handling that.